README.md

```diff
@@ -23,29 +23,31 @@ In ReactiveSwift, “cold” signals are represented by the `SignalProducer` typ
 and “hot” signals are represented by the `Signal` type.
 
 “Cold” `RACSignal`s can be converted into `SignalProducer`s using the
-`bridgedSignalProducer()` free function:
+`SignalProducer` initializer:
 
 ```swift
-func bridgedSignalProducer<T>(from signal: RACSignal<T>) -> SignalProducer<T?, NSError>
+extension SignalProducer where Error == AnyError {
+	public init<SignalValue>(_ signal: RACSignal<SignalValue>) where Value == SignalValue?
+}
 ```
 
 “Hot” `RACSignal`s cannot be directly converted into `Signal`s, because _any_
 `RACSignal` subscription could potentially involve side effects. To obtain a
 `Signal`, use `RACSignal.toSignalProducer` followed by `SignalProducer.start`,
 which will make those potential side effects explicit.
 
-For the other direction, use the `toRACSignal()` instance method.
+For the other direction, use the `bridged` instance method.
```

Comment:
(Hope you don't mind me jumping in here.)
I guess it's no longer an instance method but a property.